geomesa-kafka/geomesa-kafka-datastore/src/test/scala/org/locationtech/geomesa/kafka/utils/GeoMessageSerializerTest.scala

```diff
@@ -32,10 +32,6 @@ class GeoMessageSerializerTest extends Specification {
 
       key must not(beNull)
       value must not(beNull)
-      key.length mustEqual 10
-      key(0) mustEqual 1
-      key(1) mustEqual 'X'
-      value mustEqual Array.empty[Byte]
```

Comment:
Is it worth testing that V1 messages can still be read?

Reply:
yeah, i will add a test. i did some live tests against old topics, but didn't capture that here